packages/aws-cdk-lib/aws-ec2/lib/instance.ts

```diff
@@ -315,6 +315,22 @@ export interface InstanceProps {
    * @default false
    */
   readonly ebsOptimized?: boolean;
+
+  /**
+   * If this parameter is set to true, the instance is enabled for AWS Nitro Enclaves; otherwise, it is not enabled for AWS Nitro Enclaves.
+   *
+   * @default false.
+   */
+  readonly nitroEnclaveEnabled?: boolean;
+
+  /**
+   * If you set this parameter to true, the instance is enabled for hibernation.
+   * You can't enable hibernation and AWS Nitro Enclaves on the same instance.
+   *
+   * @default false
+   */
+  readonly hibernationConfigured?: boolean;
```

Comment:
Can we change to `hibernationEnabled` to match `enclaveEnabled`? (`HibernationOptionsProperty` in L1 has the `configured`, but ...)

Reply:
I changed the property name.